src/lib.rs

```diff
@@ -741,8 +753,25 @@ impl<T: fmt::Debug + ?Sized> Format for Debug2Format<'_, T> {
 /// This adapter disables compression and uses the `core::fmt` code on-device! You should prefer
 /// `defmt::Format` over `Display` whenever possible.
 ///
-/// Note that this always uses `{}` to format the contained value, meaning that any provided defmt
-/// display hints will be ignored.
+/// example usage:
+///
+/// ```rust
+/// # struct ExpensiveThing();
+///
+/// # impl core::fmt::Display for ExpensiveThing {
```

Comment:
I guess you forgot the `#` :) Else there is a newline.

```suggestion
/// # struct ExpensiveThing();
/// #
/// # impl core::fmt::Display for ExpensiveThing {
```